sync/src/main/java/tech/pegasys/artemis/sync/PeerSync.java

```diff
@@ -47,6 +47,8 @@
 
   private final AsyncRunner asyncRunner;
 
+  private UnsignedLong startingSlot = UnsignedLong.valueOf(0);
```

Comment:
nit: UnsignedLong.ZERO

Reply:
Starting slot should be volatile since it's accessed from multiple threads.

Reply:
fixed